File: app/src/main/java/com/motrixi/datacollection/utils/LanguageUtil.java
package com.motrixi.datacollection.utils;

import android.content.Context;

import java.util.Locale;

/**
 * author : Jason
 * date   : 2020/12/8 2:43 PM
 * desc   :
 */
public class LanguageUtil {

    public static String getLanguageInfo() {

        Locale locale = Locale.getDefault();
        String lan = locale.getLanguage();
        String country = locale.getCountry();
        return lan + "/" + country;

    }

    public static String getCountryCode(Context context){
        String countryCode = "";
        Locale locale = context.getResources().getConfiguration().locale;
        countryCode = locale.getCountry();
        return  countryCode;
    }
}

File: app/src/main/java/com/motrixi/datacollection/utils/LanguageUtil.kt
package com.motrixi.datacollection.utils

import android.content.Context
import java.util.*

/**
 * author : Jason
 *  date   : 2020/10/13 6:47 PM
 *  desc   :
 */
object LanguageUtil {

    fun getLanguageInfo(): String {

        val locale: Locale = Locale.getDefault()
        val lan = locale.language
        val country = locale.country
        return "$lan/$country"

    }

    fun getCountryCode(context: Context): String{
        var countryCode = ""
        var locale = context.resources.configuration.locale
        countryCode = locale.country
        return  countryCode
    }
}